apps/volontulo/tests/views/test_offers.py

```diff
@@ -666,16 +666,16 @@ def test_offers_join_valid_form_and_anonymous_user(self):
         response = self.client.post('/offers/volontulo-offer/{}/join'.format(
             self.offer.id
         ), {
-            'email': u'anon@example.com',
-            'phone_no': u'+42 42 42 42',
-            'fullname': u'Mister Anonymous',
-            'comments': u'Some important staff.',
+            'email': 'anon@example.com',
```

Comment:
Code duplication so maybe good idea would be to move this dictionaries to one place?